Working Tutorial with Login Page

Trinidena opened the pull request:
Added .gitignore

Trinidena committed:
Working Tutorial

Trinidena committed:
Removed Unnecessary File Block

Trinidena committed:
Updated .gitignore

Trinidena committed:
Removed Unneeded Files

Trinidena committed:
Added .suo to gitignore

Trinidena committed:
Updated Home Page to remove boilerplate text

Trinidena committed:
Testing Removal of ignored gitignore listed files

Trinidena committed:
Added Login Page
Set up with default username (admin) and password (password123)

TTSDon reviewed (approved):
Looks good. I approved.

Trinidena committed:
Login Page Checks Local DB
Before it used a hardcoded username and password value
Now it checks the local db for matching user records in the User table
If matched, then you are logged in
Also added log out button
